pkg/operation/botanist/secrets.go

```diff
@@ -164,7 +164,18 @@ func (b *Botanist) generateCertificateAuthorities(ctx context.Context) error {
 		}
 	}
 
-	return nil
+	caBundleSecret, found := b.SecretsManager.Get(v1beta1constants.SecretNameCACluster)
+	if !found {
+		return fmt.Errorf("secret %q not found", v1beta1constants.SecretNameCACluster)
+	}
+
+	return b.syncShootCredentialToGarden(
+		ctx,
+		gutil.ShootProjectSecretSuffixCACluster,
+		map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleCACluster},
+		nil,
+		map[string][]byte{secretutils.DataKeyCertificateCA: caBundleSecret.Data[secretutils.DataKeyCertificateBundle]},
```

Comment:
Nit: Should we also use `DataKeyCertificateBundle` as key for the secret in the garden?

Reply:
I went for `ca.crt` for consistency with the keys used in kubernetes secrets for the same content, e.g. service account secrets, `kube-root-ca.crt`, etc. All of these are using `ca.crt` as the key.
I also think it's quite commonly expected that these contain CA bundles and components typically read a whole bundle from these keys/files if present. Hence, I don't think `bundle.crt` is needed as a key to make this more obvious.
WDYT?